openedx/features/pakx/lms/overrides/tasks.py

```diff
@@ -127,7 +129,10 @@ def check_and_send_email_to_course_learners():
                 'language': get_user_preference(item.user, LANGUAGE_KEY),
                 'completed': grades.percent >= 1.0,
                 'status_message': "Completed" if grades.percent >= 1.0 else "Pending"}
-        if grades.percent >= 1.0:
+
+        if grades.passed and float(get_course_progress_percentage(create_dummy_request(Site.objects.get_current(),
+                                                                                       item.user),
+                                                                  text_type(item.course_id))) >= 100:
```

Comment:
why are we checking the `course_id` instead of `course_progress`?

Reply:
course_id is a required parameter for the get_course_progress_percentage function.

It is required to get progress percentage.